packages/runtime/src/createGatewayRuntime.ts

```diff
@@ -1098,12 +1122,15 @@ function isDynamicUnifiedGraphSchema(
     return false;
   }
   if (typeof schema === 'string') {
-    try {
-      // sdl
-      parse(schema);
-      return false;
-    } catch {}
+    if (isValidPath(schema)) {
+      // local file path
+      return true;
+    }
+    if (isUrl(schema)) {
+      // remote url
+      return true;
+    }
```

Comment:
URLs are valid paths too.
```suggestion
    // local file or remote url
    return isValidPath(schema);
```

Reply:
Oh good catch!